scipy/special/add_newdocs.py

```diff
@@ -1310,9 +1310,16 @@ def add_newdoc(place, name, doc):
     the amplitude of `u`.
 
     Computation is by means of the arithmetic-geometric mean algorithm,
-    except when `m` is within 1e-9 of 0 or 1.  In the latter case with `m`
-    close to 1, the approximation applies only for `phi < pi/2`.
-
+    except when `m` is within 1e-9 of 0 or 1.  
+    
+    In the latter case, when m is near 1, the functions `sn`, `cn`, and `dn`
+    are found using an approximation for `phi` given by 16.15.4 in [2]_. 
+    As this approximation is only valid when `phi < pi/2` (i.e.,
+    when `u<K`), the computation only uses 16.15.4 to find the "principal" 
```

Comment:
ReST weirdness: single backticks are for references, double backticks for code

Reply:
Can you elaborate what you mean by 'reference'? Looking at the other function docstrings, it appears that they use a single backtick for variables and math equations (sometimes with the ReST role(?) keyword :math:).  Should I change the existing text for ellpj to use double backticks?

Reply:
We updated the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts) to spell this all out. It sounds like you had the right idea.

> Should I change the existing text for ellpj to use double backticks?

Depends on where. For instance, both of the two previous lines should be double.

(I know it's been a while, so you might not be interested in this PR anymore, but I will be taking a look into getting this merged, so I thought I'd go ahead and reply to the question.)